python/paddle/tensor/math.py

```diff
@@ -6864,7 +6864,7 @@ def polygamma_(x, n, name=None):
     if n == 0:
         return digamma_(x)
     else:
-        if in_dynamic_mode():
+        if in_dynamic_or_pir_mode():
```

Comment:
polygamma_ 函数被装饰器 @inplace_apis_in_dygraph_only 修饰，只能运行在动态图模式下；而我们的 pir 属于静态图，所以这里不需要改动，复原一下吧~
```suggestion
        if in_dynamic_mode():
```

Reply:
好的 done